src/storable.rs

```diff
@@ -584,3 +605,20 @@ pub(crate) const fn bytes_to_store_size(bounds: &Bounds) -> u32 {
         4
     }
 }
+
+#[cfg(test)]
+mod unit_tests {
+    use super::*;
+
+    #[test]
+    fn test_storable_for_bool() {
+        let assert_storable = |b: bool| {
+            let bytes = b.to_bytes();
+            assert_eq!(bytes.len(), 1);
+            assert_eq!(b, bool::from_bytes(bytes));
+        };
+
+        assert_storable(true);
+        assert_storable(false);
```

Comment:
This test can be written more succinctly as follows. Notice that I assertion that `bytes.len()` is 1, as that's already included in the logic of `Storable`.

```suggestion
        assert_eq!(false, Storable::from_bytes(false.to_bytes()));
        assert_eq!(true, Storable::from_bytes(true.to_bytes()));
```